src/Umbraco.Core/Composing/TypeLoader.cs

```diff
@@ -450,7 +450,7 @@ public CachedTypeNotFoundInFileException(string message, Exception innerExceptio
         ///     The <see cref="T:System.Runtime.Serialization.StreamingContext" /> that contains contextual
         ///     information about the source or destination.
         /// </param>
-        protected CachedTypeNotFoundInFileException(SerializationInfo info, StreamingContext context)
+        private CachedTypeNotFoundInFileException(SerializationInfo info, StreamingContext context)
```

Comment:
Looks like this whole class can be removed actually.  It's `internal` and not used.